jit wald_friedman

natashawatkins opened the pull request:
* The original lecture used `UnivariateSpline` which changes the plots quite a bit...
* I don't seem to be getting the right numbers for the simulation... not sure why...

natashawatkins commented:
Fixed the second problem I was having

My results now line up with the original lecture, but only when I use `UnivariateSpline`... `interp` makes the effect of double `c` not nearly as stark

QBatista commented:
@natashawatkins I originally had the same problem. The reason is that `UnivariateSpline` with `k=1` does a poor job of interpolating as it basically fits a straight line if I remember correctly. Meanwhile, `interp` does a better job and gives results that are consistent with the Julia lecture.

natashawatkins commented:
@jstac I've updated nearly all the notation and added in the equivalent bellman operator `h`

A few things:
* Do we still want _x_ 's - or do we want to replace this with just talking about the 'true distribution'?
* I wasn't sure about this line - is this referring to linear interpolation?
![image](https://user-images.githubusercontent.com/21038129/47697085-f20e1800-dbdf-11e8-9618-de4b8f794e8d.png)
* I need to change the figure with the probabilities... will have to find the tikz somewhere...
* I was playing around with parameters and it seemed like the beta dist parameters we have from odu work well
* Should I create a new additional notebook with more comparative statics? We are currently linking to a notebook which has sliders..

jstac commented:
Thanks @natashawatkins.  Let's discuss this when you get back.
